packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -25,30 +25,20 @@ export class CoinbaseWalletProvider extends EventEmitter implements ProviderInte
   private readonly communicator: Communicator;
 
   private signer: Signer | null;
-  protected accounts: AddressString[] = [];
-  protected chain: Chain;
```

Comment:
provider won't copy the accounts/chain values anymore. they can be accessed via `signer`